docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Purpose
+### Issue dell/csi-baremetal#<!--- insert your issue number here -->
+
+_Describe your changes_
+
+## PR checklist
+- [ ] Add link to the issue
+- [ ] Choose PR label
+- [ ] New unit tests added
+- [ ] Modified code has meaningful comments
+- [ ] All TODOs are linked with the issues
+- [ ] All comments are resolved
```

Comment:
I think we should exclude "All comments are resolved" from the checklist.
PR checklist is used to signal other people that PR is ready. When all comments are resolved you're merging your changes and don't need for check it.

Reply:
Checklist is used not only to inform others but for submitter to make sure that all steps are done before merging.